change/@microsoft-teams-js-05b6fda8-1215-45ac-b8c5-338fe7d7f77e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added `customTelemetry` capability under `copilot` to send app loading data to the host. The capability is still awaiting support in one or most host applications. To track availability of this capability across different hosts see https://aka.ms/capmatrix",
```

Comment:
@TrevorJoelHarris do we update the capability matrix for private capabilities? Should we remove the link from this message?